scipy/stats/_generate_pyx.py

```diff
@@ -0,0 +1,15 @@
+import pathlib
+import numpy as np
+
+def make_biasedurn():
```

Comment:
Should the resuling `biasedurn.pyx` and `biasedurn.cxx` generated files should be added to the gitignore?

Reply:
Done in next commit.